airflow/www/templates/airflow/gantt.html

```diff
@@ -23,6 +23,8 @@
     href="{{url_for('appbuilder.static',filename='datepicker/bootstrap-datepicker.css')}}" >
 <link type="text/css" href="{{ url_for('static', filename='css/gantt.css') }}" rel="stylesheet" />
 <link type="text/css" href="{{ url_for('static', filename='css/tree.css') }}" rel="stylesheet" />
+<meta name="dag-id" content="{{ dag.dag_id }}">
```

Comment:
Likewise here - what if someone puts a double quote in the dag id (This sort of break-out was a CVE in 1.10.3 so I want to make sure we don't regress.)

Reply:
http://jinja.pocoo.org/docs/2.10/templates/#html-escaping
Currently, the JSON code contains quotes, so adding the next one does not cause problems.

We have discussed this problem in the past:
https://github.com/apache/airflow/pull/4787#discussion_r265930474

Reply:
Thanks. Bad memory, sorry.